pyperformance/tests/__init__.py

```diff
@@ -39,6 +39,10 @@ def run_cmd(cmd, *args, capture=None, onfail='exit', verbose=True):
 
     if verbose:
         print(f"(tests) Execute: {argv_str}", flush=True)
+
+    if timeout:
+        kwargs['timeout'] = 60
```

Comment:
Should we add a test that deliberately times out to confirm it's working?

Reply:
yes, it makes sense. I'll add it tomorrow.

Reply:
As the timeout is implemented in pyperf now (https://github.com/psf/pyperf/pull/205) I created a test in that PR. pyperformance now just passes the info to pyperf.
Would you like to have a test that checks the timeout is actually passed to pyperf?

Reply:
My comment was for earlier when timeout was handled here. I'm fine with not having a timeout test in pyperformance now that the actual implementation is in pyperf.